README.md

```diff
@@ -2,83 +2,40 @@
-operate the cluster to facilitate their own work
-
-**Catalog operator**: the person or team responsible for adminstration of the
-Service Catalog, including catalog curation and Service Broker registration
-
-**Broker operator**: the person or team responsible for running and managing one
-or more **Service Brokers**.
-
-**Service Producer**: the person or team who authors and/or operates a Service
-available from the Service Catalog. As part of creating a service, the Service
-Producer may also be running a Service Broker.
-
-**Resource type**: a logical Kubernetes concept. Examples include:
-
-  - [Pod](http://kubernetes.io/docs/user-guide/pods/)s
-  - [Service](http://kubernetes.io/docs/user-guide/services/)s
-  - [Secret](http://kubernetes.io/docs/user-guide/secrets/)s
-
-**Resource**: a specific instantiation of an aforementioned resource type,
-often represented as a YAML or JSON file that is submitted or retrieved via the
-standard Kubernetes API (or via `kubectl`)
-
-**Binding**: represents a relationship between an Application and a Service
-Instance. A Binding contains the information necessary for the Application to
-make use of the Service Instance.
-
-### Purpose and Scope
+## Contributing
 
-The exact purpose of the SIG will grow over time, and as such, please see
-the various "version" folders under the [docs](./docs) directory for the
-exact list of use-cases, features and design decisions that have been agreed to.
+Interested in contributing?  Check out the [documentation](./CONTRIBUTING.md)
 
-However, this SIG is (initially) focused on implementing integration with the
-[Open Service Broker API](https://github.com/openservicebrokerapi/servicebroker).
-Additionally, we will be exploring what can be done to help Service developers
-when they want to expose their code via a Service Broker.
+## Kubernetes Incubator
 
-We are currently scoping our **v1** milestone [here](./docs/v1). Interested
-in contributing?  Check out the [documentation](./CONTRIBUTING.md)
+This is a [Kubernetes Incubator project](https://github.com/kubernetes/community/blob/master/incubator.md).
+The project was established 2016-Sept-12.  The incubator team for the project is:
 
-## Contributing
+- Sponsor: Brian Grant (@bgrant0607)
+- Champion: Paul Morie (@pmorie)
+- SIG: [sig-service-catalog](https://github.com/kubernetes/community/tree/master/sig-service-catalog)
 
-We have a google mailing list: https://groups.google.com/forum/#!forum/kubernetes-sig-service-catalog
+## Code of Conduct
 
-We have a SIG Slack Channel: https://kubernetes.slack.com/archives/sig-service-catalog
+Participation in the Kubernetes community is governed by the
+[Kubernetes Code of Conduct](./code-of-conduct.md).
 
-We have a weekly call: https://plus.google.com/hangouts/_/google.com/k8s-sig-service-catalog   at 1pm PT on Mondays
-- And an agenda doc: https://docs.google.com/document/d/10VsJjstYfnqeQKCgXGgI43kQWnWFSx8JTH7wFh8CmPA/edit
```

Comment:
Agree that the details should be in k8s/community. That repo has been useful to me to find out when and where a sig is, more than once.